src/compas_rhino/scene/meshobject.py

```diff
@@ -215,7 +151,7 @@ def draw(self):
                 disjoint=self.disjoint,
             )
 
-            # geometry.Transform(transformation_to_rhino(self.worldtransformation))
+            geometry.Transform(transformation_to_rhino(self.worldtransformation))
```

Comment:
This is weird, do you know why it was commented out and why it's needed now?

Reply:
scene objects can have a transformation. before it was applied through the vertex caching. now it is done explicitly here...